imod/mf6/interfaces/imodel.py

```diff
@@ -21,4 +21,7 @@ def validate(self, model_name: str = "") -> StatusInfoBase:
     
     @property
     def domain(self):
-         raise NotImplementedError        
\ No newline at end of file
+         raise NotImplementedError        
+    
+    def is_support_regridding(self)->Tuple[bool, str]:
```

Comment:
The name of this method doesn't fully cover its purpose. It doesn't only return a boolean but also an error message.
You could rename the method to validate_regridding_supported and return a StatusInfo

Reply:
it does not return an error message, but rather the name of the first package that does not support regridding. For that reason I prefer not to return a StatusInfo. I have added a descriptive docstring.